src/components/results/feedback/index.js

```diff
@@ -0,0 +1,40 @@
+import {useState} from "react";
+import useLoadedValue from "lib/hooks/use-loaded-value";
+import useTranslate from "lib/hooks/use-translate";
+import {userCompletedFeedbackQuery} from "lib/recoil/feedback";
+import Modal from "./modal";
+import style from "./style.scss";
+
+export default function Feedback() {
+  const userCompletedFeedback = useLoadedValue(userCompletedFeedbackQuery);
+  const translate = useTranslate();
+  const [showModal, setShowModal] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+
+  if(userCompletedFeedback) { return null; }
+
+  const openModal = () => { setShowModal(true); };
+  const closeModal = ({isSubmission}) => {
+    setShowModal(false);
+    if(isSubmission) {
+      setSubmitted(true);
+    }
+  };
+
+  return (
+    !submitted && (
```

Comment:
Usually we'd indent again inside the `(`, but in this case, I think we could just move submitted to outside this statement and add `if(submitted) { return null; }` above

Reply:
updated